packages/code-du-travail-modeles/src/__test__/extract.implemented.cc.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import Engine from "publicodes";
+import { mergeModels } from "../internal/merger";
+import { extractImplementedCc } from "../internal/ExtractSupportedCc";
+
+const engine = new Engine(mergeModels());
+
+test("Extract all supported CC from the YAML modeles", () => {
+  const result = extractImplementedCc(engine);
+
+  expect(result).toContain(expect.arrayContaining([44, 176, 650]));
```

Comment:
where does these number comme from ? We have implemented more ccn

Reply:
Done :)